README.md

```diff
@@ -67,7 +67,7 @@ and if you plan to run the emulator with Docker you must use the environment var
 | `--contracts`                 | `FLOW_CONTRACTS`             | `false`        | Deploy common contracts when emulator starts                                                                                                                                                                                                        |
 | `--coverage-reporting`        | `FLOW_COVERAGEREPORTING`     | `false`        | Enable Cadence code coverage reporting                                                                                                                                                                                                        |
 | `--contract-removal`          | `FLOW_CONTRACTREMOVAL`            | `true`         | Allow removal of already deployed contracts, used for updating during development                                                                                                                                                                   |
-| `--skip-transaction-validation` | `FLOW_SKIPTRANSACTIONVALIDATION` | `false`        | Skip verification of transaction signatures and sequence numbers                                                                                                                                                                                    |
+| `--skip-tx-validation` | `FLOW_SKIPTRANSACTIONVALIDATION` | `false`        | Skip verification of transaction signatures and sequence numbers                                                                                                                                                                                    |
```

Comment:
should we also change `FLOW_SKIPTRANSACTIONVALIDATION` to  `FLOW_SKIPTXVALIDATION` ?